mysql-8.0/Dockerfile

```diff
@@ -0,0 +1,73 @@
+FROM    gitpod/workspace-full
+
+
+##
+## -- Set up the environment for a Docker build
+##    -----------------------------------------
+USER    root
+RUN     apt-get update 
+RUN     apt-get install debconf-utils -y -q
+ENV     DEBIAN_FRONTEND=noninteractive

```

Comment:
Another note, `ENV` instruction will get persisted in the image, forever. This is not what you want, because once you have a container with a Terminal you can interact with, it will still believe that the frontend is non-interactive.

The solution is to define it just for the `apt-get` commands, like so:

```Dockerfile
USER gitpod
RUN sudo apt-get update \
 && DEBIAN_FRONTEND=noninteractive sudo apt-get install -y \
    package \
 && sudo rm -rf /var/lib/apt/lists/*
```